bindings/rust/bench/src/openssl.rs

```diff
@@ -114,4 +117,33 @@ impl TlsBenchHarness for OpenSslHarness {
             .expect("Handshake not completed")
             == SslVersion::TLS1_3
     }
+
+    fn send(&mut self, sender: Mode, data: &[u8]) -> Result<(), Box<dyn Error>> {
+        let send_conn = match sender {
+            Mode::Client => &mut self.client_conn,
+            Mode::Server => &mut self.server_conn,
+        };
+
+        let mut write_offset = 0;
+        while write_offset < data.len() {
+            write_offset += send_conn.write(&data[write_offset..data.len()])?;
+            send_conn.flush()?;
```

Comment:
I'm guessing that we have the flush inside the loop because otherwise internal buffers might fill up? Might be nice to call that out in a comment.